ScnSoft-Education/LeetCodeTraining.Tests/LongestSubstring.Tests.cs

```diff
@@ -1,6 +1,5 @@
 ﻿using LeetCodeTraining.LongestSubstringWithoutRepeating3;
-
-namespace LeetCodeTraining.Tests
+namespace LongestSubstringWithoutRepeatingTests
```

Comment:
fix namespace, it should be as "LeetCodeTraining.Tests.[FileName]"

Reply:
fixed errors in namespace